e2etest/scripts/ci.sh

```diff
@@ -99,8 +99,8 @@ if [ "$PROVIDER_NAME" = "tpm" ] || [ "$PROVIDER_NAME" = "all" ]; then
     tpm_server &
     TPM_SRV_PID=$!
     sleep 5
-    tpm2_startup -c -T mssim 
-    tpm2_changeauth -c owner tpm_pass 
+    tpm2_startup -c 2>/dev/null
```

Comment:
If it's that script that the container is meant to run, you could also add the Go installation steps here? So at least you wouldn't need to build the container.
Not ideal though if you want to run the script locally.

Reply:
script gets used locally and in CI - see comment: https://github.com/parallaxsecond/parsec-client-go/pull/34#issuecomment-801906448